### DISCUSSION TITLE
Next Dev Session 19 MARCH

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1180] [General]
## All are welcome

**When:** 19 March 1930 Singapore/1230 CET/1130 GMT/0730 EST (Apologies forgot to account for daylight savings time)

**Where:** [via Matrix](https://matrix.to/#/!CAAdpmuJhdNJkxurrp:matrix.org?via=matrix.org) (The conference widget will be created 10 minutes prior)

**Agenda:** 
- Major Initiatives
     - NumFocus
     - Mesa Geo @jackiekazil 
     - Front End Visual @aiwhoo
     - Complexity Explorer Tutorial @tpike3 
- Mesa Development
     - #1173 
     - #1184
- Open Forum

@rht, @EwoutH , @jackiekazil please feel free to update, change, recommend as you see fit

**Comment by wang-boyu** [comment]
Hi! Just to check when the meeting will start in eastern time - would it be 0830 EDT instead of 0730 EST?

**Reply by aiwhoo** [reply]
I'm not good with time either. Considering Singapore doesn't follow the daylight savings time, I'd love a clarification. I'm on the east coast as well.

**Reply by jackiekazil** [reply]
Thank you for flagging! It should be 7:30am EST. We can make adjustments in the future if there were conflicts, etc, but it is difficult to make such a change the week of.

@rht, I believe that is 7:30 pm for you. Are you able to make that?

**Reply by rht** [reply]
Sounds good.

**Comment by EwoutH** [comment]
Unfortunately, I'm away for the weekend so won't be able to join

**Comment by aiwhoo** [comment]
@tpike3 My students have put together a presentation to go over some of the changes that we've done. I'll post here in case anyone may be interested: [AI Whoo Presentation Link](https://www.canva.com/design/DAE7RND3Aa4/r3JdgvcdBnSeOFB9IxQAQA/view?utm_content=DAE7RND3Aa4&utm_campaign=designshare&utm_medium=link&utm_source=sharebutton)
